stdlib/Artifacts/src/Artifacts.jl

```diff
@@ -653,6 +653,9 @@ access a single file/directory within an artifact.  Example:
 
 !!! compat "Julia 1.6"
     Slash-indexing requires at least Julia 1.6.
+
+!!! compat "Julia 1.9"
+    The argument `artifacts_toml_path` requires at least Julia 1.9.
```

Comment:
This is still not documenting the third optional argument (and the second one is not mentioned at all either).